pkg/controller/certificaterequest/certificaterequest_controller.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/aws/aws-sdk-go/service/route53"
```

Comment:
Instead we should import the local package certman-operator/awsclient instead of the proper route53 package. This will allow us to mock the client for unit tests. The awsclient already implements the ListHostedZones function so it should be a drop in replacement.